test/extensions/filters/http/transformation/inja_transformer_test.cc

```diff
@@ -637,6 +675,32 @@ TEST_F(InjaTransformerTest, DontParseBodyAndExtractFromIt) {
   EXPECT_EQ(body.toString(), "json");
 }
 
+TEST_F(InjaTransformerTest, DontParseBodyAndExtractFromReplacementText) {
+  Http::TestRequestHeaderMapImpl headers{{":method", "GET"}, {":path", "/foo"}};
+  Buffer::OwnedImpl body("not json body");
+
+  TransformationTemplate transformation;
+  transformation.set_parse_body_behavior(TransformationTemplate::DontParse);
+  transformation.set_advanced_templates(true);
```

Comment:
nit: I don't think this is necessary here? Not hurting anything, but may add confusion for future readers

Reply:
We do this in most of the surrounding tests (e.g. if you expand the file and look at `DontParseBodyAndExtractFromIt`, we do the same thing there). This affects the way we reference the extraction in the transformation definition. 

Currently, the new test, similar to the surrounding tests, references the extraction as such:
```cpp
transformation.mutable_body()->set_text("{{extraction(\"param\")}}");
```

If we disabled advanced templates, we would have to instead do
```cpp
transformation.mutable_body()->set_text("{{ param }}");
```

I think it's worthwhile to leave this as-is to maintain consistency with the existing tests in this file